pyproject.toml

```diff
@@ -74,8 +74,38 @@ force-exclude = '''
 
 [tool.ruff]
 line-length = 88
-select = ["E", "F", "RUF100", "T100", "W", "YTT"]
-extend-ignore = ["E501","E711","E721","E731","E741","F403","F821","F841"]
+select = ["ALL"]
+extend-ignore = [
+    "A001", "A002", "A003",
```

Comment:
Would it be an idea to add a comment after each what it is we are not checking? Our future selves would thank us in that we can then quickly see whether it is something we just simply do not care about or whether someone things something like "yes, that we really should do, let me remove it and fix the failures".

Reply:
I can do that. That would be helpful for future maintainers.

Reply:
I second this request.

Reply:
Ok. I added the error type, which is pretty descriptive. Also I marked which can be auto fixed and which I think should be fixed or not.